pywbem/_server.py

```diff
@@ -461,6 +482,100 @@ def create_namespace(self, namespace):
 
         return std_namespace
 
+    def delete_namespace(self, namespace):
+        """
+        Delete the specified CIM namespace in the WBEM server and
+        update this WBEMServer object to reflect the removed namespace
+        there.
+
+        The specified namespace must be empty (i.e. must not contain any
+        classes, instances, or qualifier types.
```

Comment:
is this qualifier types or qualifier declarations?

Reply:
According to DSP0004, it is qualifier types. In pywbem, we are stuck with the class name `QualifierDeclaration`, so I believe we are using both.